scripts/helmAcs.sh

```diff
@@ -147,30 +151,17 @@ EOF
     echo "REGISTRYCREDENTIALS value is empty skipping..."
   fi
 
-  # We get Bastion AZ and Region to get a valid right region and query for volumes
```

Comment:
Please try and fix the logic here for the upgrade scenario rather than reverting to passing the SOLR volume ID around as a parameter. This approach is a temporary fix for the persistence problem until we have Kubernetes "native" approach, passing the volume ID around will make it harder/more effort to remove in the future.

Reply:
Many thanks for the feedback. I will try to bring it back and see more in details why it is failing for the upgrade scenario.

Reply:
I found the problem a Volumes[?State=='available' || State=='in-use'] was missing for the update scenario where the volume is already in use.